### ISSUE TITLE
<missing alt attribute in Glasgow image>

### ISSUE THREAD

**Issue by Taha-Elmahmoudi** [CodeYourFuture/github_issues_prs_practice#135]
### Link to the page

https://codeyourfuture.github.io/github_issues_prs_practice/

### Steps to reproduce

we run lighthouse accessibility test and we found the Glasgow image does not have alt attribute
